frontend/ui/src/reducers/index.ts

```diff
@@ -5,17 +5,20 @@ import * as authActions from '../actions/user';
 import {clearUserData} from '../cookies';
 
 import data, {DataState} from './data';
+import ui, {UIState} from './ui';
 
 export * from './data';
 
 type Action = ActionType<typeof authActions>;
 
 export type StateModel = {
   data: DataState;
+  ui: UIState;
```

Comment:
This PR adds a UI reducer to make it possible to change the ui state from different parts of the app.

Reply:
What happens if I switch conversations? I.e. wouldn't the state persist so that you see the same suggestions on the message input of the other conversation?

Alternatively, this state could be handled in the `MessengerContainer`, because `MessageInput` and `MessageList` are siblings.

Reply:
Sorry, i had the feeling I forgot something 🤦 . Thanks for finding that one. Yes, it could be in the `MessageContainer`, but that would mean sending callbacks and states around. But will do that now 😉

Reply:
Ah and because my message was misleading: the overlay closes when clicking anywhere, so switching between chats would have worked flawlessly. Just the statemachine was a too much if one callback could do it.